src/coord/src/command.rs

```diff
@@ -153,6 +153,11 @@ pub enum ExecuteResponse {
     TransactionExited {
         was_implicit: bool,
         tag: &'static str,
+        /// If Some, a channel containing the results of durably storing the
```

Comment:
Does a single failure mean any other transaction operations were rolled-back? I kind of wonder if it'd be better for this message to be the final result of a transaction (failed or succeeded) instead of have an additional "still need to process more to find out if it succeeded" step. Can you add a comment either way?

Reply:
Yeah that's fair, I had been vaguely planning to deal with this as a followup, but it's probably better to bite the bullet and do it now. Lemme think through the details of what the story should be here and get back to this. Persistence already multiplexes multiple "streams" (tables, sources, etc) and individual writes are atomic so given that we already just batch up the whole txn in memory, it's definitely easiest to expose some API for an atomic write to multiple tables and use that.

We could also build support for "rollbacks" (presumably a -1 in dd but the restart recovery part gets tricky here), but I'm hoping to punt on that until at least after 1.0. Do you expect the "txn batches everything up and doesn't do anything with it until commit" model to change anytime soon? Nikhil's instinct here was no.

Reply:
No, that's not going to change anytime soon.

Reply:
Perfect. Makes this much easier